guvnor-m2repo-editor/guvnor-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/helpers/HttpPutHelper.java

```diff
@@ -53,70 +53,67 @@ public void handle( final HttpServletRequest request,
             //Get destination path
             String pathInfo = request.getPathInfo();
 
-            if ( pathInfo == null ) {
-                response.sendError( HttpServletResponse.SC_NOT_FOUND );
+            if (pathInfo == null) {
+                response.sendError(HttpServletResponse.SC_NOT_FOUND);
                 return;
             }
 
-            pathInfo = URLDecoder.decode( pathInfo,
-                                          "UTF-8" );
+            pathInfo = URLDecoder.decode(pathInfo,
+                                         "UTF-8");
+
+            String repositoryName = request.getParameter("repository");
```

Comment:
Is this guaranteed?

A ```mvn deploy``` from the CLI will issue a HTTP PUT but will not include a repository name.